core/services/fluxmonitor/helpers_test.go

```diff
@@ -94,10 +94,10 @@ func dataWithResult(t *testing.T, result decimal.Decimal) adapterResponseData {
 	return data
 }
 
-// XXXTestingOnlyCreateJob is used in TestFluxMonitorAntiSpamLogic to create a
+// CreateJob is used in TestFluxMonitorAntiSpamLogic to create a
 // job with a specific answer and round, for testing nodes with malicious
 // behavior
-func (fm *concreteFluxMonitor) XXXTestingOnlyCreateJob(t *testing.T,
+func (fm *concreteFluxMonitor) CreateJob(t *testing.T,
```

Comment:
@coventry can you shed any light on this rename?

Reply:
The `XXXTestingOnly` is a hold-over from when the simulated-blockchain test lived in `internal`. Now that it lives in `fluxmonitor`, the fact that it's in a `*_test.go` file forces that the function can only be used in tests.